core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -199,7 +199,7 @@ default ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connec
     }
 
     /**
-     * List table and view names, possibly filtered by schema. An empty list is returned if none match.
+     * List table names, possibly filtered by schema. An empty list is returned if none match.
```

Comment:
Hive connector also returns views as part of `listTables` call.
If we decide we go this way, we should add `listRelations` method too